paymentsheet/api/paymentsheet.api

```diff
@@ -5,6 +5,9 @@ public final class com/stripe/android/common/ui/ComposableSingletons$BottomSheet
 	public final fun getLambda-1$paymentsheet_release ()Lkotlin/jvm/functions/Function2;
 }
 
+public final class com/stripe/android/customersheet/CustomerAdapterResultKtxKt {
```

Comment:
I think you can get rid of this by doing a `@file:RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)`